docs/faq-and-cookbook.rst

```diff
@@ -25,3 +25,19 @@ Getting and modifying a span
 
     current_span = trace.get_current_span()
     current_span.set_attribute("hometown", "seattle")
+
+Capturing baggage at different contexts
+---------------------------------------
+
+.. code-block:: python
+
+    from opentelemetry import trace
+
+    tracer = trace.get_tracer(__name__)
+    with tracer.start_as_current_span(name="root span", parent=None) as root_span:
+        parent_ctx = baggage.set_baggage("context", "parent")
+        with tracer.start_as_current_span(name="child span", parent=root_span) as child_span:
```

Comment:
Same comment as above here, you'll want to pass in the context, rather than the span